docs/adr/0001-use-architecture-decision-records.md

```
@@ -0,0 +1,17 @@
+# 1. Use Architecture Decision Records
+
+Date: 2025-11-22
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)